react-app/src/components/ConfirmationDialog/userPrompt.test.tsx

```diff
@@ -144,7 +144,7 @@ describe("userPrompt", () => {
 
     const { children: dialogFooterChildren } = getByTestId("dialog-footer");
 
-    expect(dialogFooterChildren.item(0).textContent).toEqual("Custom Accept");
-    expect(dialogFooterChildren.item(1).textContent).toEqual("Custom Cancel");
+    expect(dialogFooterChildren.item(0)?.textContent).toEqual("Custom Accept");
+    expect(dialogFooterChildren.item(1)?.textContent).toEqual("Custom Cancel");
```

Comment:
What's the advantage of adding `?` if the test will fail when developing?

Reply:
because it's the difference between throwing an error or failing the test

Reply:
Wouldn't it be more explicit to just check whether `dialogFooterChildren` has two child nodes and then check their `textContent`?